### PR TITLE
boards: arm: bl654_usb: Fix offset load

### LINKED ISSUE #38245: board: bl654_usb project: samples/basic/blinky does not blink LED
Built and flashed the blinky sample project for the bl654_usb board.
following this guide: https://docs.zephyrproject.org/latest/boards/arm/bl654_usb/doc/bl654_usb.html

But the LED does not flash.

Tested also samples/bluetooth/hci_usb, but dongle is not detected as ble device after flashing.
About to test some serial console to see if I get any output from the dongle after flashing.

Environment:
Version: DTC 1.5.0
Python 3.8.10
cmake version 3.21.1
Zephyr version: 2.6.99
Toolchain: zephyr 0.13.0
Build OS: Linux Ubuntu 20.04 (west build, nrfutil pkg generate)
Flash: OS: Windows 10 (nrfutil dfu usb-serial)

### PR DESCRIPTION
**Pull request by nordicjm**
Fixes applications not building for offset 0x1000

Signed-off-by: Jamie McCrae <jamie.mccrae@lairdconnect.com>

Fixes https://github.com/zephyrproject-rtos/zephyr/issues/38245

### CODE REVIEW

File: boards/arm/bl654_usb/bl654_usb.dts [hunk 1]
```diff
@@ -8,8 +8,8 @@
 #include <nordic/nrf52840_qiaa.dtsi>
 
 / {
-	model = "Laird BL654 USB adapter";
-	compatible = "laird,bl654_usb";
+	model = "Laird Connectivity BL654 USB adapter";
+	compatible = "lairdconnect,bl654_usb";
```
Comment (MaureenHelm) [inline boards/arm/bl654_usb/bl654_usb.dts:12]: This change looks unrelated to the bug fix, please move it to a separate commit.

**Review by MaureenHelm — CHANGES_REQUESTED** [review]

**Review by MaureenHelm — APPROVED** [review]

**Review by cfriedt — APPROVED** [review]
